charts/steward/templates/service-monitor-run.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.metrics.serviceMonitors.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: steward-run-controller-metrics 
+  namespace: steward-system 
```

Comment:
Namespace is configurable! see `targetNamespace.name`

Reply:
Fixed with 78dadc3.